AUTHORS.md

```diff
@@ -268,3 +268,4 @@ Authors
 * [YourDaddyIsHere](https://github.com/YourDaddyIsHere)
 * [Zach Shepherd](https://github.com/zjs)
 * [陈三](https://github.com/chenxsan)
+* [Yuseong Cho](https://github.com/g6123)
```

Comment:
It looks like this should be alphabetical order (so, before Zach Shepherd).

Reply:
Oh I see. Fixed in 2ae090529e66a059e0e077d233f427583caa7c6c.